guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc

```diff
@@ -68,73 +80,92 @@ Note in particular:
 If not, select them.
 . Click the *Operating System* tab, and verify that all fields contain values.
 Confirm each aspect of the operating system.
-. Click *Resolve* in *Provisioning template* to check the new host can identify the right provisioning templates to use.
+. Click *Resolve* in *Provisioning Templates* to check the new host can identify the right provisioning templates to use.
 +
 For more information about associating provisioning templates, see xref:provisioning-templates_{context}[].
 +
-ifdef::foreman-el,foreman-deb,katello[]
+ifdef::foreman-el,foreman-deb[]
 . If you use the Katello plug-in, click the *Parameters* tab, and ensure that a parameter exists that provides an activation key.
 If not, add an activation key.
 endif::[]
```

Comment:
@ekohl @maximiliankolb Does it make sense to have this here? I thought that both `foreman-el` and `foreman-deb` are vanilla Foreman, i.e. always without Katello.
(That's why I moved `katello` to the condition below.)

Reply:
Yes.

IMHO there is no Katello on Debian, simply because the plug-in itself does not run/is not build for Debian/Ubuntu.

Second, I dislike the concept of line 87+88. This is totally (!) unrelated to your PR, but I feel like we should always drop the "if you use Katello, ..." and handle it using ifdefs aka. only show it for Katello.

Reply:
Exactly, that's what I was thinking. :D